addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -144,7 +153,7 @@ def _create_partner(self, lead_id, action, partner_id):
         #wizard and would probably diserve to be refactored or at least
         #moved to a better place
         if action == 'each_exist_or_create':
```

Comment:
This is used only in crm.lead2opportunity.partner.mass wizard
Maybe we could move this part to that wizard ?

Reply:
I have a task for that ! To be done in future commits.